webdriver-spec.html

```diff
@@ -9218,8 +9218,31 @@ <h3>Take Screenshot</h3>
 
  <li><p>When the user agent is next to <a>run the animation frame callbacks</a>:
   <ol>
-   <li><p>Let <var>root rect</var> be the <a>current top-level browsing context</a>’s
-    <a>document element</a>’s <a>rectangle</a>.
+   <li><p>Let <var>window</var> be the <a>associated window</a>
+     of the <a>current browsing context</a>’s <a>active document</a>.
```

Comment:
The original wording of this captured the visible contents of the OS window. Would `current top level browser context` be more in keeping with the spirit of this?